README.md

```diff
@@ -75,12 +88,32 @@ The directory where your Brewfile is located.
       roles:
         - geerlingguy.homebrew
 
-See the `tests/local-testing` directory for an example of running this role over Ansible's `local` connection. See also: [Mac Development Ansible Playbook](https://github.com/geerlingguy/mac-dev-playbook).
+See the `tests/local-testing` directory for an example of running this role over
+Ansible's `local` connection. See also:
+[Mac Development Ansible Playbook][mac-dev-playbook].
 
 ## License
 
-MIT / BSD
+[MIT][mit-link]
 
 ## Author Information
 
-This role was created in 2014 by [Jeff Geerling](https://www.jeffgeerling.com/), author of [Ansible for DevOps](https://www.ansiblefordevops.com/).
+This role was created in 2014 by [Jeff Geerling][author-website], author of
+[Ansible for DevOps][ansible-for-devops].
+
+#### Maintainer(s)
+
+- [Elliot Weiser](https://github.com/elliotweiser)
```

Comment:
I still want to be in the list :D — I just haven't been as active of late since I haven't bought a new Mac recently.